php/libraries/TimePoint.class.inc

```diff
@@ -224,12 +227,15 @@ class TimePoint
         $VisitLabel = $nextVisitLabel == null
             ? 'V' . $row['nextVisitNo']
             : $nextVisitLabel;
+        $centerID   = $psc == null
+        ? $row['CenterID']
+        : $psc;
```

Comment:
I don't think there's much reason to use the ternary operator here, and if/else would do fine.